auth-api/tests/unit/services/test_entity.py

```diff
@@ -168,3 +168,28 @@ def test_get_contact_by_business_identifier_no_contact(session):  # pylint:disab
     factory_entity_model(business_identifier='CP1234567')
     contact = EntityService.get_contact_for_business('CP1234567')
     assert contact is None
+
+
+def entity_factory(business_identifier, pass_code):
+    """Factory."""
+    return EntityModel(business_identifier=business_identifier, pass_code=pass_code)
+
+
+def test_validate_pass_code(app, session):  # pylint:disable=unused-argument
+    """Assert that a contact for an entity can be updated."""
+    entity_info = {
+        'businessIdentifier': 'CP12345678',
+        'pass_code': '12345678'
+    }
+    with app.app_context():
```

Comment:
Not a big deal, but you don't actually need the `with app.app_context()` here.  Including `session` as a parameter for the test is sufficient.

Reply:
removed.